### PR TITLE
Upgrading ruby, node and rails version

### PR DESCRIPTION
**Pull request by alaxalves**
Fixes part of https://github.com/publiclab/spectral-workbench/issues/469

Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- `rake test`
* [x] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [x] pull request are descriptively named
* [x] if possible, multiple commits squashed if they're smaller changes
* [ ] reviewed/confirmed/tested by another contributor or maintainer

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

**Comment by gr455** [comment]
Wow great!!

File: Gemfile [hunk 1]
```diff
@@ -1,14 +1,10 @@
...
 #source 'https://rails-assets.org'
 
-ruby '~>2.1.2'
-gem 'rails', '~>3.2'
+ruby '~>2.6.5'
+gem 'rails', '~>4.2.8'
 
 # Database handling
-group :sqlite do
-  gem 'sqlite3'
```
Comment (gr455) [inline Gemfile]: Aren't we using sqlite for production?

Comment (alaxalves) [reply Gemfile]: No, SQLite is not used for production purposes, even if Spectral was a low-traffic website. See: https://stackoverflow.com/questions/913067/sqlite-as-a-production-database-for-a-low-traffic-site

Comment (gr455) [reply Gemfile]: Okay cool :+1:

**Pull request merged**